src/helpers/approvals-satisfied.ts

```diff
@@ -65,4 +65,14 @@ export const approvalsSatisfied = async ({ teams, number_of_reviewers = '1', pul
   return booleans.every(Boolean);
 };
 
-const createArtificialCodeOwnersEntry = (teams: string[]) => teams.map(team => ({ owners: [team] }));
+const createArtificialCodeOwnersEntry = (teams?: string[], users?: string[]) => [{ owners: (teams || []).concat(users || []) }];
+const distinct = (arrayWithDuplicates: string[]) => arrayWithDuplicates.filter((n, i) => arrayWithDuplicates.indexOf(n) === i);
+const isTeam = (teamOrUser: string) => teamOrUser.includes('/');
```

Comment:
I don't think this is always a safe assumption, since you can pass teams without the full organization path. Could we handle teams and users separately?

Reply:
Afaik, teams have to have full path which includes org

Also we cannot handle separately because for manual inputs we can still write separate code, but for codeowners it won't work. So we need a solution which works for both manual input and codeowners file

I believe this will

We can add it in the doc that you should use full path for teams which include org. What say?

Reply:
I'm saying when you build the `loginsLists` list, rather than iterating over teams and users and checking "is team", why not build each list separately and concatenate at the end

Reply:
Yes we can do this, but what if the inputs are not provided, and the flow uses codeowners file and within codeowners file users are provided instead of teams?

Reply:
As per discussion, this check is fine since CODEOWNERS expect full team name, and in case of teams in input we are updating and validating it